modules/tinymce/src/core/main/ts/api/Resource.ts

```diff
@@ -9,8 +9,8 @@ import ScriptLoader from './dom/ScriptLoader';
 import Promise from './util/Promise';
 
 interface Resource {
-  load(id: string, url: string): Promise<any>;
-  add(id: string, data: any): void;
+  load <T = any>(id: string, url: string): Promise<T>;
+  add (id: string, data: any): void;
```

Comment:
Was wondering if this should match up with the `add` function below?

Reply:
Oops, I originally changed both and then reverted it. I'll revert the one below 👍